### ISSUE TITLE
Error copying vendor and/or third party libraries.

### ISSUE THREAD

**Issue by expleosoftware** [madhusudhand/angular2-cli#8]
OSX El Capitan (10.11.6)
Creating an app and trying to start a server:

```
$node -v 
v5.12.0

$ng2 -V
0.2.3

$ng2 create TestApp
App created.

$cd TestApp

$ng2 init
Installed npm packages. (49 sec)

$ng2 serve
Clean successful.
Error copying vendor and/or third party libraries.
Unhandled rejection undefined
```

same for ng2 serve --dev, ng2 serve --prod, ng2 build...

Looks like issue is in angular-build.js file
module.exports = {
  vendorNpmFiles: [
    // these will be copied to dist/vendor
    '@angular',
    'rxjs', **<-- after fresh installation this package is named as rx in node_modules**
    'systemjs',
    'zone.js',
    'reflect-metadata'
  ],
  otherFiles: [
    // add any files that needs to be copied to dist or any given path
    // valid values
    //   'src/CNAME'                        -> this copies file to dist folder
    //   { src: 'src/CNAME', dest: 'dist'}  -> this copies file to the given location
    'src/CNAME'
  ]
};

**npm install rxjs@5.0.0-beta.6 --save did the job for me**

**Comment by madhusudhand** [comment]
Can you list the dependencies from package.json and npm version.

**Comment by expleosoftware** [comment]
```
$ npm -v
3.8.6
```

That's fresh package.json content after ng2 init:

{
  "name": "TestApp",
  "version": "0.0.0",
  "license": "MIT",
  "private": true,
  "readme": "README.md",
  "keywords": [
    "angular",
    "angular2-cli"
  ],
  "angular2-cli": {},
  "scripts": {
    "lint": "tslint \"src/*_/_.ts\"",
    "test": "karma start karma.conf.js"
  },
  "dependencies": {
    "@angular/common": "^2.0.0-rc.4",
    "@angular/compiler": "^2.0.0-rc.4",
    "@angular/core": "^2.0.0-rc.4",
    "@angular/http": "^2.0.0-rc.4",
    "@angular/platform-browser": "^2.0.0-rc.4",
    "@angular/platform-browser-dynamic": "^2.0.0-rc.4",
    "@angular/router": "^0.2.0",
    "reflect-metadata": "0.1.3",
    "systemjs": "^0.19.36",
    "zone.js": "^0.6.12"
  },
  "devDependencies": {
    "angular2-cli": "^0.2.3",
    "codelyzer": "0.0.26",
    "jasmine-core": "^2.4.1",
    "jasmine-spec-reporter": "^2.5.0",
    "karma": "^1.1.2",
    "karma-chrome-launcher": "^1.0.1",
    "karma-jasmine": "^1.0.2",
    "tslint": "^3.14.0"
  }
}

**Comment by madhusudhand** [comment]
Issue observed with npm 3. Also there are many performance issues have been observed with npm 3.
will be working on these.
@expleosoftware try with npm 2.15 to start working with cli.

**Comment by madhusudhand** [comment]
#9

**Comment by madhusudhand** [comment]
fix is made in angular2-cli@0.4.0
